build.gradle

```diff
@@ -148,3 +148,27 @@ configurations.all {
     resolutionStrategy.force 'commons-codec:commons-codec:1.13'
     resolutionStrategy.force 'com.google.guava:guava:31.0.1-jre'
 }
+
+// versionIncrement: Task to auto increment to the next development iteration
+task versionIncrement {
+    onlyIf { System.getProperty('newVersion') }
+    doLast {
+        ext.newVersion = System.getProperty('newVersion')
+        println "Setting version to ${newVersion}."
+        // String tokenization to support -SNAPSHOT
+        ant.replaceregexp(match: opensearch_version.tokenize('-')[0], replace: newVersion.tokenize('-')[0], flags:'g', byline:true) {
+            fileset(dir: projectDir) {
+                // Include the required files that needs to be updated with new Version
+                include(name: ".github/workflows/sql-workbench-test-and-build-workflow.yml")
+                include(name: "build.gradle")
```

Comment:
Thanks, I can add more restrictive parsing, will push in my latest commit.

Reply:
Can you please check now @joshuali925 
Sample PR with new change: https://github.com/prudhvigodithi/sql/pull/2
Does this sample PR for version increment looks good?

Reply:
approved, last question: i see it's hard coded to use snapshot in build.gradle `+ '-SNAPSHOT"'`, will there be a case we use `x.x.x` instead of `x.x.x-SNAPSHOT`?